python/fastscore/suite/connect.py

```diff
@@ -254,7 +254,7 @@ def get(self, name):
             xx = self.swg.connect_get(self.name, name=name)
         except Exception as e:
             raise FastScoreError("Cannot retrieve '%s' instance info" % name, caused_by=e)
-        if len(xx) > 0 and xx[0].health == 'ok':
+        if ((len(xx) > 0 and xx[0].health == 'ok') or (skipUnhealthy == False)):
```

Comment:
I think this needs to be:
if len(xx) > 0 and (xx[0].health == 'ok' or skipUnhealthy == False):